tools/bin/make/airbyte_ci_check.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
`set -o errexit -o nounset -o pipefail` in all bash scripts.

Consider running these through the shellcheck linter.